px-map-behavior-popup.es6.js

```diff
@@ -247,8 +247,9 @@
 
       const maxWidth = 400;
       const minWidth = 300;
+      const maxHeight = 300;
```

Comment:
Yes, I will take out the maxHeight, let's move with the double click event for now.